clang/lib/CIR/Dialect/IR/CIRTypes.cpp

```diff
@@ -13,6 +13,7 @@
 #include "clang/CIR/Dialect/IR/CIRTypes.h"
 
 #include "mlir/IR/DialectImplementation.h"
+#include "mlir/Support/LLVM.h"
```

Comment:
Why is this necessary?

Reply:
Good catch, I missed that it was added by my IDE, it is unnecessary, I will remove it.